packages/language-server/test/tooltip-spec.ts

```diff
@@ -0,0 +1,188 @@
+import { expect } from "chai";
+import { getHoverResponse } from "../src/hover-tooltip";
+import { UI5SemanticModel } from "@ui5-language-assistant/semantic-model-types";
+import {
+  generateModel,
+  expectExists,
+} from "@ui5-language-assistant/test-utils";
+import {
+  TextDocument,
+  Position,
+  Hover,
+  TextDocumentIdentifier,
+  TextDocumentPositionParams,
+} from "vscode-languageserver";
+
+describe.only("the UI5 language assistant Hover Tooltip Service", () => {
```

Comment:
Add tests for open/close tag:
- Open tag with namespace
- Close tag with namespace (same as open tag)
- Close tag without namespace and no default namespace defined (should return `undefined`)
- Close tag with namespace which is is different from open tag namespace (should use the close tag namespace)
- Close tag without namespace when the open tag has a namespace (should use the default namespace)

Reply:
It can be in the tooltip package (doesn't have to be in language-server)